meterfs/meterfs.h

```diff
@@ -29,7 +29,7 @@ typedef struct {
 			size_t sectors;
 			size_t filesz;
 			size_t recordsz;
-			char name[8];
+			char name[9];
```

Comment:
If you introduce new define MAX_NAME_LEN use it here (MAX_NAME_LEN + 1) instead of magic number. Other aproach is to use sizeof(name) - 1 instead of define.

Reply:
done